dashboard/app/controllers/courses_controller.rb

```diff
@@ -173,6 +173,11 @@ def check_plc_enrollment
   end
 
   def render_no_access
+    unless @unit_group.can_be_instructor?(current_user) || @unit_group.can_be_participant?(current_user)
```

Comment:
Would it make sense to replace the check here and in scripts_controller.rb with CanCan?

Reply:
Updated!